### PR TITLE
STR-49 -- FE: Verify a users session inactivity

### PR DESCRIPTION
**Pull request by ekno27**
# Description
'Automatically logs out user after 27 minutes of inactivity per CSUN's standard'
  
# Testing
1. add the .env variable i sent in my email to your .env file
2. change the variable's value to something like .125 (.125 = ~7 seconds after conversion)
2. login
3. interact with the page in some small way: move the mouse around, click, etc
3. Do nothing in the window: do not move mouse, touch keyboard, scroll wheel, etc.
4. wait
5. You should be logged out after the time specified in the env value

### CODE REVIEW

**Review by liadGolan — DISMISSED** [review]
👍

**Review by frlzjosh — CHANGES_REQUESTED** [review]
Unable to logout from the navbar component

File: resources/js/App.vue [hunk 1]
```diff
@@ -1,33 +1,60 @@
...
+          this.$router.push({ name: 'welcome' })
         }
       } else {
         this.$router.push({ name: 'Login' })
       }
+    },
+    checkUserInactivity() {
+      var idleTimeout= parseInt(document.querySelector('meta[name=idle-timeout]').content)  
+      this.timeout = setTimeout(()=>{
+        console.log('logout')
```
Comment (frlzjosh) [inline resources/js/App.vue]: do we still need the console.log ?

**Review by frlzjosh — CHANGES_REQUESTED** [review]
One some request

File: resources/js/App.vue [hunk 2]
```diff
@@ -1,33 +1,59 @@
...
   methods: {
+    ...mapActions([
+      'clearUserData'
+    ]),
     checkForCookies () {
       if (document.cookie.includes('userKey')) {
         if (this.$route.fullPath === '/login') {
-          this.$router.push({ name: 'Dashboard' })
+          this.$router.push({ name: 'welcome' })
         }
```
Comment (frlzjosh) [inline resources/js/App.vue:31]: you chose the wrong path when fixing merge conflicts

**Review by frlzjosh — CHANGES_REQUESTED** [review]
one small lil change

**Review by frlzjosh — APPROVED** [review]
Is goood

**Review by liadGolan — CHANGES_REQUESTED** [review]
Log out does not reflect until I move my mouse after the timeout. Also the .env change needs to be reflected in .env.example

**Review by liadGolan — APPROVED** [review]

**Review by andrewMETALAB — APPROVED** [review]
👍

**Pull request merged**